config/helpers.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const fs = require('fs');
+
+const localConfigFileName = 'config.dev.json';
+const startRemoteClientSecret = 'bWlncmF0aW9ucy5vcGVuc2hpZnQuaW8K';
+
+const getLocalConfig = () => {
+  const configPath = path.join(__dirname, localConfigFileName);
+  if (!fs.existsSync(configPath)) {
+    console.error('ERROR: config/config.dev.json is missing');
+    console.error(
+      'Copy config/config.dev.json.example to config/config.dev.json' +
+        ' and optionally configure your dev settings. A valid clusterUrl is ' +
+        ' required for start:remote.'
+    );
+    process.exit(1);
+  }
+  const localConfig = require(configPath);
+  const migMeta = {
```

Comment:
Let's call it `virtMeta` everywhere instead of `migMeta`

Reply:
I actually wonder if we even need to create a meta object here at all. All the values are just copied from the JSON file which we already have parsed as `localConfig`. Why don't we just call the config.dev.json file `virtMeta.dev.json` instead, and have it structured the way we want this object structured (oauth stuff nested), and use it directly (just `return require(configPath)` and then in other places `const virtMeta = getLocalConfig()`).

Reply:
done